docs/shared_bindings_matrix.py

```diff
@@ -191,6 +213,29 @@ def get_excluded_boards(base):
                         base[module]["excluded"][board_chip].append(entry.name)
                     else:
                         base[module]["excluded"][board_chip] = [entry.name]
+
+            for module in check_dependent_modules:
+                depend_results = set()
+
+                parents = check_dependent_modules[module].split("CIRCUITPY_")
+                parents = [item.strip(", ").lower() for item in parents if item]
+
+                for parent in parents:
+                    if parent in base:
+                        if (board_chip in base[parent]["excluded"] and
+                            entry.name in base[parent]["excluded"][board_chip]):
+                                depend_results.add(False)
+                        else:
+                            depend_results.add(True)
+
+                # only exclude the module if there were zero parents enabled
+                # as determined by the 'depend_results' set.
+                if (len(depend_results) == 1 and False in depend_results):
```

Comment:
Why restrict on the length here? `all()` (all True) and `any()` (any True) can do truthiness of iterable values.

Reply:
Never used either, so I didn't even think about them. Makes much more sense. Thanks!